tests/package-title-dir-rtl-001/OPS/content_001.xhtml

```diff
@@ -0,0 +1,18 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
```

Comment:
Nit: remove blank lines and adjust indentation to match https://github.com/w3c/epub-tests/blob/main/tests/xx-epub-template/OPS/content_001.xhtml?

I changed it a few PRs ago.

Reply:
Done it in https://github.com/w3c/epub-tests/pull/31/commits/dc9f0d396bd4d814812b3d707225bef4f0f59357